portfolio/src/main/webapp/script.js

```diff
@@ -61,9 +61,20 @@ window.onload = () => {
 /* 
  * Get message from server and insert it on the home page.
  */
-window.onload = () => {
-  fetch('/data').then(response => {
-    // response.text() is the server message.
-    document.getElementById('message-container').innerText = response.text();
+getServerMessage = () => {
+  fetch('/data').then(response => response.text()).then(message => {
+    document.getElementById('message-container').innerText = message;
   });
 }
+
+window.onload = () => {
```

Comment:
nit: add function comment on this too.

comments generally needed on any 'public'/global fns.